examples/fields.go

```diff
@@ -6,16 +6,28 @@ import (
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/nimbleos"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/service"
+	"os"
 )
 
+func checkEnvironmentVariableExists() {
+	if os.Getenv("SDK_TARGET_HOST") == "" ||
+		os.Getenv("SDK_TARGET_USER") == "" ||
+		os.Getenv("SDK_TARGET_USER_PASSWORD") == "" {
+		fmt.Println("ERROR: Missing one of these environment variables: SDK_TARGET_HOST, SDK_TARGET_USER, SDK_TARGET_USER_PASSWORD")
+		os.Exit(1)
+	}
+}
+
 func main() {
+	checkEnvironmentVariableExists()
```

Comment:
I suggest reading the env vars just once.  For example:

	host := os.Getenv("SDK_TARGET_HOST")
	user := os.Getenv("SDK_TARGET_USER")
	password := os.Getenv("SDK_TARGET_PASSWORD")

	if host == "" || user == "" || password == "" {
		fmt.Println("ERROR: These environment variables must be defined: SDK_TARGET_HOST, SDK_TARGET_USER, SDK_TARGET_PASSWORD.")
		os.Exit(1)
	}

Then use the same variables below.